CMakeLists.txt

```diff
@@ -251,14 +251,15 @@ int main(void)
 {
   void *trace[1];
   int trace_size = backtrace(trace, 1);
+  (void)trace_size;
```

Comment:
Hmm.. I've thought just calling `backtrace` here would be good/better - but then the return value is not checked maybe?

Reply:
Yes, clang gives warning of dead assignment here, this is no-op use if the variable should suppress it. This is the trick @justinmk taught me

Reply:
Yeah, but wouldn't just calling `backtrace(trace, 1);` also do it?

Reply:
it complains about `trace_size` var

Reply:
Yes, and that would not need to be defined (I guess).
The test is only to see if `backtrace` is available.

Reply:
The test will still fail if backtrace is not available. But this also create clang warning and @justinmk agreed that it is better to fix this rather than putting an ignore on the test.

Reply:
What I mean is that this will also avoid the warning (since there is not unused var in the first place):
```diff
diff --git i/CMakeLists.txt w/CMakeLists.txt
index 947de6198..3d3c19efe 100644
--- i/CMakeLists.txt
+++ w/CMakeLists.txt
@@ -250,7 +250,7 @@ check_c_source_compiles("
 int main(void)
 {
   void *trace[1];
-  int trace_size = backtrace(trace, 1);
+  backtrace(trace, 1);
   return 0;
 }
 " HAVE_EXECINFO_BACKTRACE)
```

Reply:
oh... sorry, I did not get that. Yes, absolutely!